k8/deployment.yaml

```diff
@@ -9,7 +9,6 @@ metadata:
   name: polaris-deployment
   namespace: polaris
 spec:
-  replicas: 2
```

Comment:
Minor suggestion: can we put `1` explicitly in case people don't know the default value?

Reply:
Took care.